types/sitemap-generator/sitemap-generator-tests.ts

```diff
@@ -22,18 +22,20 @@ const generator = SitemapGenerator('https://example.com/', {
     wait: () => {},
 });
 
-generator.on('add', () => {});
+generator.on('add', (url: string) => {
+    url.trim();
+});
 generator.on('done', () => {});
-generator.on('ignore', () => {});
+generator.on('ignore', (url: string) => {
+    url.trim();
+});
```

Comment:
```suggestion
});
generator.on('ignore', () => {});
```
same, it verifies change is backward compatible